ActiveSong/src/org/jjazz/activesong/Bundle_fr_FR.properties

```diff
@@ -0,0 +1,3 @@
+OpenIDE-Module-Name=ActiveSong
+ErrSongIsPlaying=Impossible d'active ce projet tant qu'un autre est en lecture.
```

Comment:
Il semble y avoir une coquille ici :
> Impossible d'active ce projet tant qu'un autre est en lecture.
devrait probabalement être :
> Impossible d'active**r** ce projet tant qu'un autre est en lecture.

Reply:
Une aide extérieure quelle bonne surprise !!! Merci beaucoup Olivier. J'ai corrigé directement depuis Crowdin. Si jamais tu veux aider pour les traductions voici le lien: https://crwd.in/jjazzlab       C'est vraiment beaucoup plus pratique, tout se synchronise automatiquement avec GitHub etc.